### DISCUSSION TITLE
It's been several years and brewed x265 is still missing NEON support 🤣

### DISCUSSION THREAD

**Discussion by Jerry1144** [Homebrew/discussions#5164] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.10
ORIGIN: https://mirrors.tuna.tsinghua.edu.cn/git/homebrew/brew.git
HEAD: c6d959218f143cd17b1fc3e0f10f143cbd273528
Last commit: 3 days ago
Core tap JSON: 27 Feb 13:05 UTC
Core cask tap JSON: 27 Feb 13:05 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_API_DOMAIN: https://mirrors.tuna.tsinghua.edu.cn/homebrew-bottles/api
HOMEBREW_BOTTLE_DOMAIN: https://mirrors.tuna.tsinghua.edu.cn/homebrew-bottles
HOMEBREW_BREW_GIT_REMOTE: https://mirrors.tuna.tsinghua.edu.cn/git/homebrew/brew.git
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://mirrors.tuna.tsinghua.edu.cn/git/homebrew/homebrew-core.git
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: dodeca-core 64-bit arm_lobos
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3.1-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I'm quite surprised to see `x265 [info]: using cpu capabilities: none!` in 2024🤣. Almost panicked as I thought there's major problem with brew's bottles, but then I found `x264` using ARMv8 and NEON no problem.

The previous attempt to enable NEON in `libx265` was https://github.com/Homebrew/homebrew-core/pull/91840. Yes, it's one year since that, and some 3 years since x265 3.5 release. The whole arm64 code is post-3.5 so it'd be major effort trying to cherrypick HEAD patches into a 3.5_1, although there's still no x265 3.6 in sight.

Figured it'd be rude to start an Issue for this. Anyways, for future selves' reference, I post it here.

**Comment by SMillerDev** [comment, marked as answer]
The title is incorrect, a better one would be:
> It's been several years and stable x265 is still missing NEON support 🤣

Which should be directed at the x265 developers who have not cut a release with it yet.
